ntoskrnl/inbv/inbv.c

```diff
@@ -861,6 +878,7 @@ INIT_FUNCTION
 InbvRotBarInit(VOID)
 {
     PltRotBarStatus = 1;
+    // TODO: Possibly do other initializations
```

Comment:
Here I was wondering whether we could move the initialization of the RotBar temp buffer (with Inbv/VidScreenToBufferBlt calls) here... Just a thought!

Reply:
This would require moving all SKU-detection-related code :)